docker-mender-dist-packages

```diff
@@ -78,3 +78,16 @@ docker run --rm \
         ${MENDER_CONFIGURE_VERSION:-master} \
         amd64 \
         ${CI_PIPELINE_ID:-LOCAL}
+
+# mender-monitor
+if [ -n "${MENDER_TEST_BOT_ACCESS_TOKEN}" ]; then
```

Comment:
Custom token added only to the `test-monitor-debian` branch in GitLab for now, but this (I believe), will be required for the protected branches henceforth.

Reply:
Yes, that is correct and is a prize we need to pay in the name of security.

The missing detail then is that the new acceptance tests will only be able to run on protected branches, so we need a decorator to skip it on PRs. (Still a bit rusty from my vacation, so I can't think about a good example for this, but we do have in other test suites...)